Plugins/Public/base_plugin/PlayerCommands.cpp

```diff
@@ -59,8 +59,11 @@ namespace PlayerCommands
 		pages[1] = L"<TRA bold=\"true\"/><TEXT>/bank withdraw [credits], /bank deposit [credits], /bank status</TEXT><TRA bold=\"false\"/><PARA/>"
 			L"<TEXT>Withdraw, deposit or check the status of the credits held by the base's bank.</TEXT><PARA/><PARA/>"
 
-			L"<TRA bold=\"true\"/><TEXT>/shop price [item] [price] [min stock] [max stock]</TEXT><TRA bold=\"false\"/><PARA/>"
-			L"<TEXT>Set the [price] of [item]. If the current stock is less than [min stock]"
+			L"<TRA bold=\"true\"/><TEXT>/shop price [item] [price]</TEXT><TRA bold=\"false\"/><PARA/>"
+			L"<TEXT>Set the [price] of [item].</TEXT><PARA/><PARA/>"
+
+			L"<TRA bold=\"true\"/><TEXT>/shop stock [item] [min stock] [max stock]</TEXT><TRA bold=\"false\"/><PARA/>"
+			L"<TEXT>Set the [min stock] and [max stock] of [item]. If the current stock is less than [min stock]"

```

Comment:
For these menus it is worth checking that they never go over 4000 characters. If they go over 4096 the XML becomes borked for reasons I am not entirely sure of (likely a client side buffer issue). Issue should be fixed in FLHook 4.2, but for the time being a check will do for Disco